media/js/products/vpn/affiliate-attribution.es6.js

```diff
@@ -199,6 +199,11 @@ AffiliateAttribution.init = function () {
 
         FxaProductButton.init()
             .then((flowParams) => {
+                if (!flowParams) {
+                    reject('FxA flow params are undefined.');
```

Comment:
Weirdly, I can only see this message in the console on Firefox. Safari and Chrome don't show the reject message, but just locally console logging I can see they fall into the correct condition and there are no 500 errors so this looks good to me.

Reply:
Hmm, great spot! I did a little searching and it seems promises will swallow errors like this. Even throwing an error in a `catch()` will just convert to a rejection of the promise.

Changing this to a simple `console.error()` seems to do the trick cross browser. I added a commit if you want to test it out?

Reply:
TIL. It shows in Safari for me now but Chrome is still silent 🤷‍♀️. I think seeing the error in Fx and Safari is an improvement though, so happy to stick with `console.error` here. (I don't see anything in my Chrome inspector or settings that should block the error message, but it's probably not worth too much debugging time)

Reply:
Hmm, works in Chrome for me? 🤷‍♂️